features/all-positions/LiquidationActionDialog.tsx

```diff
@@ -0,0 +1,488 @@
+                      liquidatedPosition.liquidationReceipt
+                    )}
+                    target="_blank"
+                  >
+                    {prettyAddress(liquidatedPosition.liquidationReceipt)}
+                  </a>
+                </Status>
+                <Status>
+                  <Label>Liquidation ID: </Label>
+                  {liquidatedPosition.liquidationId}
+                </Status>
+                <Status>
+                  <Label>Tokens liquidated: </Label>
+                  {prettyBalance(
+                    Number(liquidatedPosition.tokensLiquidated)
+                  )}{" "}
+                  {tokenSymbol}
+                </Status>
+                <Status>
+                  <Label>Locked Collateral: </Label>
+                  {prettyBalance(
+                    Number(liquidatedPosition.lockedCollateral)
+                  )}{" "}
+                  {collSymbol}
+                </Status>
+                <Status>
+                  <Label>
+                    {invertDisputablePrice ? `Minimum` : `Maximum`} Disputable
+                    Price:{" "}
+                  </Label>
+
+                  {prettyBalance(
+                    getDisputablePrice(liquidatedPosition.maxDisputablePrice)
+                  )}
+                </Status>
+                <hr />
+                <Box pt={2}>
+                  <Typography>
+                    <strong>Dispute this liquidation</strong>
+                    <br></br>
+                  </Typography>
+                  {translateLiquidationStatus(
+                    Number(liquidatedPosition.liquidationTimestamp),
+                    liquidatedPosition.status
+                  ).status !== "Liquidation Pending" && (
+                    <span>
+                      <Typography>
+                        This liquidation can't be disputed as it is no longer in
+                        the pending state as it has passed the liquidation
+                        liveness period.
+                      </Typography>
+                      <Box textAlign="center" pt={3} pb={2}>
+                        <Button fullWidth variant="contained" disabled>
+                          Dispute Liquidation
+                        </Button>
+                      </Box>
+                    </span>
+                  )}
+                  {translateLiquidationStatus(
+                    Number(liquidatedPosition.liquidationTimestamp),
+                    liquidatedPosition.status
+                  ).status === "Liquidation Pending" && (
+                    <span>
+                      <Typography>
+                        This liquidation is currently in it's liveness period
+                        and can be disputed if you think it was invalid. In
+                        order for this liquidation to be invalid, the identifier{" "}
+                        {priceIdUtf8} would need to be greater than{" "}
```

Comment:
"greater" if not `invertedPrice`, but "less" if `invertedPrice`

Reply:
done.

Reply:
I don't see the changes?